packages/microsoft_kiota_serialization_json/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+### Bug Fixes
```

Comment:
I'm not sure I understand why you edited the changelog manually?

Reply:
Oh I wasn't sure if I was supposed to. I'll revert this then